data/remnant missions.txt

```diff
@@ -1716,3 +1720,164 @@ mission "Remnant: Broken Jump Drive job"
 		dialog "The research team quickly offloads the broken jump drive and pays you <payment> before hurrying back to their lab."
 
 
+
+mission "Remnant: Face to Maw 1"
+	name "Talk to Plume about the void sprites"
+	description "Plume would be very interested in a report on what happened with the void sprites."
+	landing
+	to offer
+		has "Remnant: Return the Samples 2: done"
```

Comment:
Would it make sense to offer this mission only a few weeks after the sample mission chain? The first lines of dialog feel kinda weird if the player just returned to Nenia immediately

Reply:
Hmmm. You're right. It would fit better if there was at least a short delay. Might also keep players guessing about where else they should try heading back to in the future. I guess I'll need to make an event timer.

Reply:
OK, event timer set for 14 days added.